cairo/src/StandardOutcome/AssetOutcome.cairo

```diff
@@ -0,0 +1,13 @@
+# Q - does a felt work as a destination?
+
+struct AllocationEntry:
+    member destination: felt
+    member amount: felt
+    member data: felt 
+end
```

Comment:
We will need to align our terminology at some early opportunity; on my branch I had been playing with some other names.